### DISCUSSION TITLE
Run PostgreSQL with Custom Configuration

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#61] [KubeDB]
# Run PostgreSQL with Custom Configuration

KubeDB simplifies Provision, Upgrade, Scaling, Volume Expansion, Monitor, Backup, Restore for various Databases in Kubernetes on any Public & Private Cloud

https://kubedb.com/docs/v2023.12.11/guides/postgres/configuration/using-config-file/

**Comment by manju4ever** [comment]
How to place a custom pg_hba.conf just like postgres.conf ?

**Reply by amartich** [reply]
The bitnami package has some documentation about it. Not sure it'd work with kubeDB.
